scrapers/StudioFOW.yml

```diff
@@ -9,29 +13,31 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //img[@class="rev-slidebg"]/@title
+      Title: //h2[contains(@class,"woodmart-title-container") and not(contains(text(),"DOWNLOADS"))]/text()
       Details:
-        selector: //div[contains(@class,"woodmart-title-container")]//p
+        selector: //div[contains(@class,"woodmart-title-container")]//text()
+        postProcess:
+          - replace:
+            - regex: ^\s+$
+              with: ""
```

Comment:
Was there something wrong with the existing details?  For a few scenes i tested it seemed ok